src/components/templates/Input/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+
+const Input = ({
+  name,
+  defaultValue = '',
+  style,
+  className,
+  placeholder = '정보를 입력하세요.',
+  value,
+  onChange,
+}) => {
+  const inputStyle = {
+    width: '324px',
+    height: '40px',
+    color: 'black',
+    border: '1px solid #ebebeb',
+    fontSize: '18px',
+    padding: '1% 2%',
+  }
+
+  return (
+    <input
+      type="text"
+      name={name}
+      value={value}
+      style={{ ...style, ...inputStyle }}
+      className={className}
+      placeholder={placeholder}
+      onChange={() => onChange}
```

Comment:
```suggestion
      onChange={(onChange}
```

컨벤션 차이지만 위의 사항대로 적어주시는거 어떠신가요? 😊